compiler-plugin/src/main/scala/scalapb/compiler/ProtoValidation.scala

```diff
@@ -103,10 +103,17 @@ class ProtoValidation(implicits: DescriptorImplicits) {
           s"${m.getFullName}: sealed oneofs may not contain nested enums"
         )
       }
-    } else if (m.sealedOneOfExtendsCount > 0) {
-      throw new GeneratorException(
-        s"${m.getFullName}: is not a Sealed oneof and may not contain a sealed_oneof_extends message option. Use extends instead."
-      )
+    } else {
+      if (m.sealedOneOfExtendsCount > 0) {
+        throw new GeneratorException(
+          s"${m.getFullName} uses `sealed_oneof_extends` option while not being a sealed oneof (see https://scalapb.github.io/docs/sealed-oneofs). Did you mean `extends`?"
```

Comment:
@thesamet what do you think about this phrasing?

Reply:
I'd prefer: `${m.fullName}: sealed_one_of_extend can only be used on sealed one ofs. Did you mean `extends`?".

Reply:
Done